lib/streamlit/hashing.py

```diff
@@ -406,6 +407,12 @@ def _to_bytes(self, obj, context):
             self.update(h, obj.tell())
             return h.digest()
 
+        elif isinstance(obj, Pattern):
```

Comment:
Huh, I didn't think this would work... I thought the `typing` module was supposed to be used only for type hints.

But either way, I think it would be clearer to say `isinstance(obj, re.Pattern)` instead.

Reply:
Tried that initially but comparing to `typing.Pattern` seems to be the correct way to go about things.
```
>>> re.Pattern
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: module 're' has no attribute 'Pattern'
```